### ISSUE TITLE
Clarify process

### ISSUE THREAD

**Issue by Ada1000** [LibCrowds/libcrowds#151]
Initially not clear whether you submit each title individually or whether you submit the whole task consisting of all titles.

Not clear whether to include all forms of entertainment or just plays.

Not clear whether to include more information than the title if the leading article is included in the announcement text, or whether to cut off the article. E.g.
"Will present a new comedy called The <linefeed>
Way to get Married"

**Comment by alexandermendes** [comment]
These are more good examples of things we need to improve in out tutorial/FAQ text.

- We're going for all selections on a page, then submit. Hopefully this will be a bit more clear once the 'Confirm on Submit' function is added (see https://github.com/LibCrowds/vue-pybossa-frontend/issues/150).

- To include only plays or other performances: we have actually discussed this one quite a lot and I'm still unsure on the choice of language. Should we call them plays, performances, events etc., and do we want to deal with only the plays for now or the other 'performances'. There are so many strange things in there - a performance by a dog was spotted recently - that we decided to keep things 'simple' and focus on just the plays for now. We might need to have another look at this.

- In the end this isn't hugely important as the selections are primarily so that people know what to transcribe later (although we might also try to use them for OCR training). But again, pointing out anything unclear is all really useful to help us write our tutorial text.

**Comment by corined** [comment]
It's also not clear if you want the punctuation included such as final full stop or quotation marks. 

What is the point of this text on the front page "see The *Way To Get Married*; or perhaps you may just prefer to see *The Romp*? There’s something of the Kent coast in *Ways and Means*; or *A Trip to Dover*."

**Comment by alexandermendes** [comment]
Again, in terms of drawing the selection boxes, it's not really important as these essentially just lead into a subsequent set of tasks where we ask people to transcribe the marked titles. It's a way of breaking down the huge amount of info on the page into far smaller tasks. In the transcription tasks we are currently keeping things simple and asking people to just type what they see. We can always run a cleanup process later if we decide we don't want punctuation etc.

It's a difficult balance between writing guidance that is thorough enough for those that think of these things and that which gets incredibly complex and potentially intimidating for others (only include the semi-colon if it preceeds an alternative title, including exactly one space after the semi-colon and only capitalising the next word if it's a proper name...and so on).

I'm not sure I know what you mean by 'the point' of the text...I suppose to give people a sense of what they might discover while contributing to the project, the full text being:

> Mark the titles of plays in historic playbills. Look out for playbills for comedies and farces; see The *Way To Get Married*; or perhaps you may just prefer to see *The Romp*? There’s something of the Kent coast in *Ways and Means*; or *A Trip to Dover*.

I was considering changing these to a more direct description of the task objective (and probably prefer that option)!

**Comment by corined** [comment]
Ok, thanks. Re: text. I don't understand the point of including that text and would agree that changing it to a more direct description of the task objective would be better.

**Comment by alexandermendes** [comment]
It gets my vote! We'll have a chat about this on our end. I think it basically came from trying to squeeze some contextual text onto that page but it does seem to confuse things a bit.

**Comment by mialondon** [comment]
@alexandermendes we need to sort out the category descriptions cos they're not really working - one of the artefacts of changing architectures, so exactly the kind of issue that we were hoping would surface in asking for feedback. If we can have more than 100 characters in the description it'd be a lot easier.

**Comment by mialondon** [comment]
@alexandermendes how much text can we include in the task description?

**Comment by mialondon** [comment]
Thanks @Ada1000 @corined - I've submitted a pull request (https://github.com/LibCrowds/project-iiif-annotate/pull/40 i.e. made some changes that could be updated onto the live site) with some tweaks based on your comments. This is exactly the useful kinds of feedback we were hoping for, so thanks for taking the time to respond!

**Comment by alexandermendes** [comment]
@mialondon so the category descriptions were intended to be a per volume description (category = volume, project = collection of tasks). We're now trying to force them in to the project descriptions just to make use of the text, which I'm not really convinced by. I think these should basically be the task objectives - "draw rectangles around the titles etc.", rather than "Look our for title x".

You cou can have 255 characters in the project descriptions. No limit on the category descriptions.

**Comment by alexandermendes** [comment]
Also remember that we will have a tutorial that we can fill with as much detail as we like. I'm getting a bit wary of including every detail in those task objectives. It now says don't include song titles but what about comedic interludes, or addresses of thanks? If we include everything here it's going to quickly fill the screen!

**Comment by mialondon** [comment]
@alexandermendes information always invisible on the screen is a gazillion* times more accessible than information in a tutorial or help screen. Usability testing will help us work out the minimum text needed to cover most situations and what text can be delegated to help for those who really need the details. *technical term

If we can sort out how volume descriptions display then that puts less emphasis on the project descriptions, but similarly, anything hidden behind a mouseover is much less likely to be seen. The model is still https://transcription.si.edu/browse?filter=all but we're not there yet!

**Comment by alexandermendes** [comment]
Ok, so a couple of weeks ago we did have the category descriptions always appearing at the top of the screen but we changed this to bring the project list further up to the top (see https://github.com/LibCrowds/vue-pybossa-frontend/issues/110).

The new model is to show category descriptions like this:

![untitled](https://user-images.githubusercontent.com/5636273/29664866-68628880-88c9-11e7-8313-34f4a7b83bb2.png)

So how should we display them now?

Remember that the difference with that Smithsonian model is that they only have one project per volume, whereas we (potentially) have many, which is one of the core reasons we redesigned this whole thing, to be able to bundle things up by category, then specific task sets.

We will have a browse all feature soon.
